platform/pc/Makefile

```diff
@@ -386,7 +393,7 @@ PCI_BUS=	pci.0
 endif
 QEMU_TAP=	-netdev tap,id=n0,ifname=tap0,script=no,downscript=no
 QEMU_NET=	-device $(NETWORK)$(NETWORK_BUS),mac=7e:b8:7e:87:4a:ea,netdev=n0 $(QEMU_TAP)
-QEMU_USERNET=	-device $(NETWORK)$(NETWORK_BUS),netdev=n0 -netdev user,id=n0,hostfwd=tcp::8080-:8080,hostfwd=tcp::9090-:9090,hostfwd=udp::5309-:5309
+QEMU_USERNET=	-device $(NETWORK)$(NETWORK_BUS),netdev=n0 -netdev user,id=n0,hostfwd=tcp::8080-:8080,hostfwd=tcp::9090-:9090,hostfwd=tcp::9091-:9091,hostfwd=udp::5309-:5309
```

Comment:
Since tracelog is exclusive with ftrace, should we just reuse the same port number?

Reply:
Ok, I changed all the instances of 9091 with 9090. There isn't strictly a reason that tracelog and ftrace couldn't be used together, but it kind of seems counterintuitive to do function profiling while having debug tracing output enabled.